tests/lib/rules/component-name-in-template-casing.js

```diff
@@ -168,6 +168,24 @@ tester.run('component-name-in-template-casing', rule, {
           <keep-alive />
         </template>
       `
+    },
+
+    // globals
+    {
+      code: `
+        <template>
+          <RouterView />
+        </template>
+      `,
+      options: ['PascalCase', { globals: ['RouterView'] }]
+    },
+    {
+      code: `
+        <template>
+          <RouterView />
+        </template>
+      `,
+      options: ['PascalCase', { globals: ['router-view'] }]
```

Comment:
Please add two test cases for the `kebab-case` option as well.

Reply:
Done.